### DISCUSSION TITLE
Podman Desktop Icon?

### DISCUSSION THREAD

**Discussion by TomSweeneyRedHat** [podman-desktop/podman-desktop#638] [Ideas]
Hi All, 

On the main https://github.com/containers page, we have the icons of the four or five most prominent projects in containers displayed near the top for easy access.  I think it would be good to do so for Podman Desktop too, and if you don't have one already, @mairin would probably be happy to assist.

Once acquired, please ping @ashley-cui and I'm sure she'd be happy to add it to the page for you.

Thanks!

t

**Comment by dylanmtaylor** [comment]
I don't own the rights to this or anything, but this appears to be the logo: https://podman-desktop.io/img/logo.svg

**Comment by mairin** [comment]
Ooh sorry I missed this one, I will prep one for the page today.

**Comment by TomSweeneyRedHat** [comment]
@mairin were you able to prep the logo for the containers page?  @benoitf could you confirm that the logo @dylanmtaylor pointed at is the correct one please?

**Reply by mairin** [reply]
@TomSweeneyRedHat I did and then didn't upload it! Here is what I did:
![PodmanDesktop_containersorg](https://user-images.githubusercontent.com/799683/197825479-89257530-1745-4549-9724-3b36a23cc64c.png)

The treatment is such that it should hang well with the others on the containers org page.

**Reply by ashley-cui** [reply]
Thanks @mairin! Feel free to open add the logo to this repo and open a PR here: https://github.com/containers/common/tree/main/logos

After that's merged, I can edit the readme

**Reply by ashley-cui** [reply]
Or I can open the PR, but I want to make sure you get credit for your work :)

**Reply by benoitf** [reply]
I created https://github.com/containers/common/pull/1212 on behalf of the image submitted here by @mairin

**Reply by mairin** [reply]
Sorry that I'm slow to the draw, thank you so much @benoitf !!

**Reply by benoitf** [reply]
https://github.com/containers/.github/pull/4 but it's rendering not as the others
